modin/core/execution/ray/common/task_wrapper.py

```diff
@@ -18,11 +18,15 @@
 """
 
 import asyncio
-
+from modin.config import LogMode
+from modin.logging import get_worker_logger, ClassLogger
+from modin.logging.config import JOB_ID
 import ray
 
+__WORKER_ENV__ = {"env_vars": {"MODIN_LOG_MODE": LogMode.get()}}
```

Comment:
please add a note to the docs about "set before running, cannot change later" behaviour